src/App.css

```diff
@@ -1,3 +1,61 @@
+html, body, #root, #root>div {
+  height: 100%
+}
+
+.searchResultsPage {
+  display: flex;
+  min-height: 100%;
+  flex-direction: column;
+  background-color: #e5e5e5;
```

Comment:
Consider replacing CSS with SASS to improve maintainability. 
SASS allows you to declare variables in CSS, so that specific colours can be re-used and changed more easily through multiple styles.

This is fine for now, but continuing in this fashion will certainly make UI maintenance more difficult later on.